### ISSUE TITLE
Compatibility with Boom?

### ISSUE THREAD

**Issue by sammarcus** [kyleneideck/BackgroundMusic#21]
Hi,
Does this work while using [Boom](http://www.globaldelight.com/boom/index.php) at the same time, or can only one of these be used simultaneously?

Thanks!

**Comment by rohapa** [comment]
I don't think it does the same thing as Boom 2. Boom 2 has an equalizer and enhancements to Spotify, that Spotify doesn't have built in already.
